VAMobile/src/components/PhotoPreview.test.tsx

```diff
@@ -1,32 +1,27 @@
-import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance } from 'react-test-renderer'
 import Mock = jest.Mock
-
-import { context, render, RenderAPI } from 'testUtils'
+import { context, render, screen } from 'testUtils'
 import PhotoPreview from 'components/PhotoPreview'
 import { Asset } from 'react-native-image-picker/src/types'
+import { bytesToFinalSizeDisplay } from 'utils/common'
+import { t } from 'i18next'
 
 context('PhotoPreview', () => {
-  let component: RenderAPI
-  let testInstance: ReactTestInstance
   let onPressSpy: Mock
   let image: Asset = {
     uri: 'testing',
     fileSize: 1234,
   }
 
   beforeEach(() => {
-    onPressSpy = jest.fn(() => {})
-
-    component = render(<PhotoPreview width={110} height={110} image={image} onDeleteCallback={onPressSpy} />)
+    onPressSpy = jest.fn(() => { })
 
-    testInstance = component.UNSAFE_root
+    render(<PhotoPreview width={110} height={110} image={image} onDeleteCallback={onPressSpy} />)
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
+  it('renders correctly', () => {
+    expect(screen.getByAccessibilityHint('Remove this photo')).toBeTruthy()
+    expect(screen.getByText(bytesToFinalSizeDisplay(1234, t, false))).toBeTruthy()
+    expect(screen.UNSAFE_getByProps({ width: 110, height: 110 })).toBeTruthy()
```

Comment:
should remove this one as it is unsafe in nature and we are trying to remove those. We also shouldn't be testing for display styling